src/resources/universalJavaApplicationStub.sh

```diff
@@ -0,0 +1,397 @@
+#!/bin/sh
```

Comment:
I really like that idea - and didn't even think of it.  I'll update the PR.  Thanks!